go/teams/rpc_exim.go

```diff
@@ -0,0 +1,40 @@
+// Copyright 2015 Keybase, Inc. All rights reserved. Use of
+// this source code is governed by the included BSD license.
+
+// Export-Import for RPC for Teams
+
+package teams
+
+import (
+	keybase1 "github.com/keybase/client/go/protocol/keybase1"
+)
+
+func (t *Team) ExportToTeamPlusAllKeys(idTime keybase1.Time) keybase1.TeamPlusAllKeys {
+	var perTeamKeys = make(map[int]keybase1.PerTeamKey)
+	var err error
+	var i = 0
+	for err == nil {
+		perTeamKeys[i], err = t.Chain.GetPerTeamKeyAtGeneration(i)
+		i++
+	}
```

Comment:
I think the first key generation is `1`. Is that right @mlsteele? So this loop might quit before it does any work.

I might feel more comfortable reaching into the private map and ranging over its keys directly. Then we don't have to worry at all about where the keys start, or whether they have holes in them.